index.md

```diff
@@ -20,6 +20,7 @@ title: Testing
 
 1.  [Basics of Testing](01-basics.html)
 2.  [Assertions](02-assertions.html)
+2a. [Design By Contract](02a-dbc.html)
```

Comment:
FWIW, I felt the rendering of 2a on the same line as 2 _was_ correct.

And, part of my issue in placing it at same levels as others was that it forces a re-numbering of everything there which I felt was intrusive (for you) not to mention tedious (for me).